### ISSUE TITLE
dplsh get keeps on reading the same data on get operation

### ISSUE THREAD

**Issue by mvwieringen** [scality/Droplet#224]
With the current libdroplet and dplsh you can put a file to an S3 server (scality S3 docker image)
but when you read your data back using the get operation it keeps on reading the same data
over and over again. I have seen the same behavior with my own code reading data via the VFS
interface it seems the EOF marker is ignored.

**Comment by mvwieringen** [comment]
b.t.w. putting and getting blobs using dpl_fput() and dpl_fget() works as expected.

**Comment by mvwieringen** [comment]
Ok had some time to debug this further. Added support to dplsh for the new stream mode
of libdroplet using dpl_fstream_get() but that runs into exactly the same problems. Then I tested
against AWS and there is works. So it seems to be a problem with libdroplet/dplsh against the
Scality S3 server running in a local docker container. The range seems to be set to outside
the actual file size but the Scality S3 server just returns the data at the beginning of the range e.g. 0
